index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Middle</title>
+</head>
+<body>
+    <section class="work">
+        <div class="work-title">
+            <h1 class="work-title-h1">My Recent Works</h1>
+            
+        </div>
+        <div class="work-items">
+            <hr>
+        </div>
+        <div class="work-items">
+            <img src="./ImgPlaceholder.png" alt="">
+            <h2>
+                Multi-Post Stories
+            </h2>
+            <p class="work-items-p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <div class="work-items-button">
+                <button>css</button>
+                <button>html</button>
+                <button>bootstrap</button>
+                <button>Ruby</button>
+            </div>
+            <div class="work-items-see">
+                <button>See Project</button>
```

Comment:
- [ ] For all buttons, follow the button styling i.e Active, Hover and Pressed (see design)